### DISCUSSION TITLE
Feature: Show wikimedia commons/wikidata images more prominent

### DISCUSSION THREAD

**Discussion by NotSoImportant** [osmandapp/OsmAnd#11558] [Ideas]
# 🚀 feature request

### Description

1. Also show `wikimedia_commons` image for **buildings** without a poi
    - Probably the same or related to #11414
1. Show image (`wikimedia_commons` or through `wikidata`) also **nearby**
    - in the "online photos" box (like done for Mapillary images, like showing up when exactly hitting the item)
    - optional: link them to the object they are coming from
1. Possibility to **show all items on the map** that have `wikimedia_commons` or `wikidata` with an image or category set.
    - could be similar to `configure map > street-level imagery`
    - like this you can also virtually explore a place

### Thematically linked feature requests
- #4942 [REQUEST] Mapillary FOSS alternative 
- #9970 Seamlessly integrate FOSS image/photo contribution

**Comment by MPeti1** [comment]
Why was this moved to a discussion?

**Reply by vshcherb** [reply]
It's not decided how it's going to be progressed and it's also contains refs to other issues
